### PR TITLE
Schedule component

### PR DESCRIPTION
**Pull request by w-gao**
Problem
=======
Schedule component [Jira](https://cruzhacks-dev.atlassian.net/browse/C2D-74)

Solution
========
What I/we did to solve this problem
* Create Schedule component

Change Summary:
---------------
* Schedule component layout on desktop d18b166
* Responsiveness 3e8559f

Dev-Ops
=======
If you added an ENV variable, please check off that you've updated the following  
- [ ] Key & Sample value to `.env` & `sample.env` 
- [ ] GCP Secret Manager (Both development and production)
- [ ] Github Secrets (Added a development and production variable)
- [ ] Github Workflows `.github/workflows/dev.yml` & `.github/workflows/prod.yml`
- [ ] webpack-config.js

Images/Important Notes (optional):
-----------------------

* This should probably be merged after #152 and #155; I need to fix the background gradient

* Desktop

* Tablet

* Mobile

  - NOTE: should ask design about the arrows on mobile

### CODE REVIEW

**Review by Alperdec — CHANGES_REQUESTED** [review]
Looks really good William! The only comments I have is comparing to figma there seems to be a little more padding between the text and the table fields and the background gradient differences. It looks like we'll have to update the gradient for the FAQ component as well.

File: src/views/Home/components/Schedule/components/index.scss [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    color: #d3ddff;
+    font-size: 20px;
+    width: 100%;
+
+    @include responsiveness.screen(smallest, phone, tablet) {
+      font-size: 16px;
+    }
+
+    &__time {
+      margin: 0;
```
Comment (krishnakpandian) [inline src/views/Home/components/Schedule/components/index.scss]: On desktop, I think the margin should be a little bit higher according to the figma

File: src/views/Home/components/Schedule/components/index.scss [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+      @include responsiveness.screen(smallest) {
+        padding: 8px 5px;
+        width: 75px;
+        min-width: 75px;
+      }
+    }
+
+    &__text {
+      margin: 0;
```
Comment (krishnakpandian) [inline src/views/Home/components/Schedule/components/index.scss]: Same as time, for desktop, the margin should be a bit higher

File: src/views/Home/components/Schedule/components/index.scss [hunk 3]
```diff
@@ -0,0 +1,118 @@
+@use "./stylesheets/responsiveness";
+@use "./stylesheets/typography";
+
+.schedule-table {
+  position: relative;
+  width: 100%;
+
+  &__table {
+    min-height: 300px;
```
Comment (krishnakpandian) [inline src/views/Home/components/Schedule/components/index.scss]: From the figma, it seems like the table should have the same height for each component and the min-height should be a bit higher on desktop

**Review by krishnakpandian — CHANGES_REQUESTED** [review]
This looks great! I agree with @Alperdec comments. I think the main thing is that on desktop the vertical sizing is a bit off but overall it looks good.

**Comment by w-gao** [comment]
Thanks for the review @Alperdec and @krishnakpandian! I adjusted the vertical spacing and made each component the same height, so it should look more like the figma design now.

File: src/views/Home/components/Schedule/components/index.scss [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+    min-height: 500px;
+    height: 500px;
+  }
+
+  &__event {
+    display: flex;
+    flex-wrap: nowrap;
+    border-bottom: 2px solid #9fc5ff;
+    color: #d3ddff;
+    font-size: 20px;
```
Comment (krishnakpandian) [inline src/views/Home/components/Schedule/components/index.scss:61]: Font-size should be closer to 30px on desktop + widescreens. We can use our font macro or screen macro for a quick fix.

Comment (krishnakpandian) [inline src/views/Home/components/Schedule/components/index.scss:61]: Font-size should be closer to 30px on desktop + widescreens. We can use our font macro or screen macro for a quick fix.

File: src/views/Home/components/Schedule/components/index.scss [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+    width: 100%;
+    height: 100%;
+    max-height: 60px;
```
Comment (krishnakpandian) [inline src/views/Home/components/Schedule/components/index.scss:64]: This seems like it should be closer to 80px on desktop + widescreen.

File: src/views/Home/components/Schedule/components/index.scss [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+      font-size: 20px;
+    }
+  }
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+    min-height: 500px;
+    height: 500px;
```
Comment (krishnakpandian) [inline src/views/Home/components/Schedule/components/index.scss:49]: These seem closer to 800px for desktop.

**Review by krishnakpandian — CHANGES_REQUESTED** [review]
I think we are almost there, it still seems a bit rectangular in comparison to the figma for desktop views. The mobile views looks amazing!

**Review by krishnakpandian — APPROVED** [review]
LGTM

**Pull request merged**